data/content/agent_attributes.yaml

```diff
@@ -127,6 +127,12 @@ attributes:
   required: false
   desc: |
       The directory to use for agent hooks. See [Hook locations](https://buildkite.com/docs/agent/v3/hooks#hook-locations) for details.
+- name: job-log-path
+  env_var: BUILDKITE_JOB_LOG_PATH
+  default_value: "platform's TempDir; e.g. '/tmp'"
+  required: false
+  desc: |
+      The directory of the temporary job log file created by the `enable-job-log-tmpfile` [agent configuration option](/docs/agent/v3/configuration). Introduced in [TBD](https://github.com/buildkite/agent/releases/tag/TBD).
```

Comment:
I left the release version as TBD as afaict this hasn't yet been released. Just looking for a review on the wording for now

Reply:
TBD = v3.50.0

Reply:
The wording here looks good though!